Makefile

```diff
@@ -1,7 +1,10 @@
 requirements:
 	pip install -r requirements.txt
 
-Run:
+build:
+	docker build -t app:0.1 --load .
```

Comment:
it's better to have a dynamic tag, we can get a tag from the hash commit
`git rev-parse --short HEAD`

Reply:
I added the dynamic tag.